PasswordRulesSharp.Cli/PasswordRulesSharp.Cli.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net8.0</TargetFramework>
+        <Nullable>enable</Nullable>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Spectre.Console.Cli" Version="0.48.1-preview.0.1" />
```

Comment:
Any reason why preview?

Reply:
I don't recall. (Possibly related to trimming support?) Upgraded it.